cosmwasm/submit-proposal.js

```diff
@@ -176,13 +176,11 @@ const registerItsChain = async (client, wallet, config, options) => {
         const chainConfig = getChainConfig(config, chain);
         const { maxUintBits, maxDecimalsWhenTruncating } = getChainTruncationParams(config, chainConfig);
 
-        const itsEdgeContract = getItsEdgeContract(chainConfig);
-
         return {
             chain: chainConfig.axelarId,
-            its_edge_contract: itsEdgeContract,
+            its_edge_contract: itsEdgeContract(chainConfig),
```

Comment:
`itsEdgeContract` might return `undefined`. Create a `tryItsEdgeContract` helper that doesn't throw an error for the usage in `itsEdgeContracts`, but use `itsEdgeContract` here that does throw an error?